.tekton/telco-core-rds-4-19-push.yaml

```diff
@@ -192,6 +192,8 @@ spec:
       params:
       - name: input
         value: $(params.prefetch-input)
+      - name: dev-package-managers 
+        value: "true"
```

Comment:
I think the commits can be a bit more organized, in which the first adds the tekton files and the second adds the Hermetic flow enablement (including prefetch & hermetic builds params). I'm even inclined to have both changes as one commit.